hybrasyl/World.cs

```diff
@@ -1652,6 +1652,49 @@ where searchTerm.IsMatch(amap.Key)
                         }
                     }
                         break;
+
+                    case "/rollchar":
+                    {
+                        // /rollchar <class> <level>
+                        // Allows you to "roll a new character" of the desired Class and Level, to see his resulting HP and MP.
+

```

Comment:
This command should require privileges, see examples for other cases in this same file.